packages/cli/test/utils/runUtils.ts

```diff
@@ -17,12 +17,20 @@ export function findApiToken(dirpath: string): string {
 }
 
 export function expectDeepEquals<T>(a: T, b: T, message: string): void {
-  expect(a).deep.equals(b, message);
+  try {
+    expect(a).toEqual(b);
+  } catch (e) {
+    expect.fail(message);
+  }
 }
 
 /**
  * Similar to `expectDeepEquals` but only checks presence of all elements in array, irrespective of their order.
  */
 export function expectDeepEqualsUnordered<T>(a: T[], b: T[], message: string): void {
-  expect(a).to.have.deep.members(b, message);
+  try {
+    expect(a.sort()).toEqual(b.sort());
```

Comment:
@nazarhussain  I don't think this works as expected, see [failed test run](https://github.com/ChainSafe/lodestar/actions/runs/7261446669/job/19782733198#step:6:4638).

Note that tests might pass sometimes, there was a previous issue related to that
- https://github.com/ChainSafe/lodestar/pull/5641

Also noticing from that issue, our test output is not really detailed compared to previous output if tests fail (see screenshot in linked issue). Previously the output showed exactly what value was wrong while now it does tell me anything about what part of the data is wrong